src/app/public/modules/split-view/split-view-drawer.component.ts

```diff
@@ -44,9 +47,15 @@ let skySplitViewNextId = 0;
 })
 export class SkySplitViewDrawerComponent implements AfterViewInit, OnInit, OnDestroy {
 
+  /**
+   * Sets the list panel's `aria-label` attribute to support accessibility.
+   */
   @Input()
   public ariaLabel: string;
 
+  /**
```

Comment:
This is missing the default value. The current docs show the default value as `320`.